api/examples/src/message/claimValue.ts

```diff
@@ -0,0 +1,39 @@
+import { GearApi, GearKeyring, UserMessageRead } from '@gear-js/api';
+
+const main = async () => {
+  const api = await GearApi.create();
+
+  const alice = await GearKeyring.fromSuri('//Alice');
+
+  const mailbox = await api.mailbox.read(GearKeyring.decodeAddress(alice.address));
+
+  const messageWithValue = mailbox.find(([_, message]) => parseInt(message.value) > 0);
+
+  console.log(messageWithValue);
+
+  api.mailbox.claimValue.submit(messageWithValue[0][1]);
+  try {
+    await new Promise((resolve, reject) => {
+      api.mailbox.claimValue.signAndSend(alice, ({ events, status }) => {
+        console.log(`STATUS: ${status.toString()}`);
+        if (status.isFinalized) resolve(status.asFinalized);
+        events.forEach(({ event }) => {
+          if (event.method === 'UserMessageRead') {
+            console.log(event.toHuman());
+          } else if (event.method === 'ExtrinsicFailed') {
```

Comment:
What do think about this way?

Instead of using `else if` just add another `if`

```js
 if (event.method === 'UserMessageRead') {
        console.log(event.toHuman());
  }
  
  if (event.method === 'ExtrinsicFailed') {
      reject(api.getExtrinsicFailedError(event).docs.join('/n'))
  }